app/models/course/group.rb

```diff
@@ -20,4 +21,16 @@ def set_defaults
     return if !course.course_users.exists?(user: creator) || group_users.exists?(user: creator)
     group_users.build(user: creator, role: :manager, creator: creator, updater: updater)
   end
+
+  def remove_duplicate_users
```

Comment:
I can only use `uniq` on new records( I don't want persisted records been affected), have updated the code, is it better now ?